App skeleton.

shoumikhin opened the pull request:
Summary: .

Differential Revision: D62281653

guangy10 commented:
FYI, review is happening in the Diff, to consolidate comments in one place, let's keep the comments there. CC: @huydhn @shoumikhin

shoumikhin committed:
App skeleton. (#5128)

Summary: Pull Request resolved: https://github.com/pytorch/executorch/pull/5128

Differential Revision: D62281653
